src/favorites/FavoriteCategoryButton.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import Icon from '../widgets/Icon';
 
-const FavoriteCategoryButton = ({ name, onClick, isFavorited }) => {
+const FavoriteCategoryButton = ({ name, onClick, isFavorited, noTitle }) => {
   return (
-    <h2 className="mt-1 mb-0 text-xs-center">
```

Comment:
Actually we have some h2 inside h1 tag, IMO FavoriteCategoryButton should not have styling by default and title. You could rewrite it like this:
```
import React from 'react';
import Icon from '../widgets/Icon';

const FavoriteCategoryButton = ({ onClick, isFavorited }) =>
  <a onClick={onClick}>
    <Icon name={isFavorited ? 'star' : 'star_border'} sm />
  </a>;

export default FavoriteCategoryButton;

```